src/items/itemsTable.component.tsx

```diff
@@ -182,24 +183,32 @@ export function ItemsTable(props: ItemTableProps) {
     React.useState<MRT_ColumnFiltersState>([]);
 
   const table = useMaterialReactTable({
-    columns: columns, // If dense only show the name column
+    columns: dense
+      ? [
+          { ...columns[0], size: 400 },
+          { ...columns[4], size: 400 },
+          { ...columns[5], size: 400 },
+          { ...columns[6], size: 400 },
+        ]
+      : columns, // If dense only show the name column
```

Comment:
This comment doesn't make sense anymore, also not quite sure on the sizes here
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/680f9cc0-ab3b-40c2-891a-54d10d0bfc28) as they take up lots of room.
It would be nice if they could scale, but I can also have a look when its on the systems page.